boards/sim/sim/sim/configs/ostest/defconfig

```diff
@@ -23,12 +23,14 @@ CONFIG_INIT_ENTRYPOINT="ostest_main"
 CONFIG_MM_KASAN=y
 CONFIG_MM_UBSAN=y
 CONFIG_MM_UBSAN_TRAP_ON_ERROR=y
+CONFIG_PRIORITY_INHERITANCE=y
 CONFIG_PTHREAD_CLEANUP=y
 CONFIG_PTHREAD_CLEANUP_STACKSIZE=3
 CONFIG_PTHREAD_MUTEX_TYPES=y
 CONFIG_RAM_START=0x00000000
 CONFIG_SCHED_HAVE_PARENT=y
 CONFIG_SCHED_WAITPID=y
+CONFIG_SIM_WALLTIME_SIGNAL=y
```

Comment:
This is needed to unblock https://github.com/apache/nuttx-apps/pull/1711 due to https://github.com/apache/nuttx-apps/issues/1716